CHANGES.rst

```diff
@@ -5,7 +5,8 @@
 2.0.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Store and register ``ISiteMapping`` site names in lowercase to
```

Comment:
It would be good if it would mention this only applies to the ZCML directive. None of the implementation is changed. In particular, anything having to do with existing (persistent) registrations is unchanged.